test/server/server_test.cc

```diff
@@ -1024,33 +1024,6 @@ TEST_P(ServerInstanceImplTest, BootstrapNodeWithOptionsOverride) {
   EXPECT_EQ("bootstrap_sub_zone", server_->localInfo().node().locality().sub_zone());
 }
 
-// Validate server runtime is parsed from bootstrap and that we can read from
-// service cluster specified disk-based overrides.
-TEST_P(ServerInstanceImplTest, BootstrapRuntime) {
-  options_.service_cluster_name_ = "some_service";
-  initialize("test/server/test_data/server/runtime_bootstrap.yaml");
-  EXPECT_FALSE(server_->enableReusePortDefault());
-  EXPECT_EQ("bar", server_->runtime().snapshot().get("foo").value().get());
-  // This should access via the override/some_service overlay.
-  EXPECT_EQ("fozz", server_->runtime().snapshot().get("fizz").value().get());
-}
-
-// Validate that a runtime absent an admin layer will fail mutating operations
-// but still support inspection of runtime values.
-TEST_P(ServerInstanceImplTest, RuntimeNoAdminLayer) {
```

Comment:
ditto, also don't think this test is `server_->enableReusePortDefault()` specific

Reply:
> ditto, also don't think this test is `server_->enableReusePortDefault()` specific.

Yeah, I also think these tests should  be kept, and just removing the runtime feature setting in runtime_bootstrap.